ui/raidboss/data/03-hw/dungeon/fractal_continuum.txt

```diff
@@ -0,0 +1,156 @@
+hideall "--reset--"
+hideall "--sync--"
+hideall "--start--"
+
+0 "--reset--" sync /:Exhibit level III is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The high-level incubation bay is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The reality augmentation bay is no longer sealed/ window 10000 jump 0
+
+
+###Phantom Ray
+#-ii F7D
+
+0 "Start"
+0.0 "--start--" sync /Exhibit level III will be sealed off/ window 0,1
+10.3 "Rapid Sever" sync /:Phantom Ray:F7A:/
+16.6 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+21.4 "Double Sever 1" sync /:Phantom Ray:F7B:/
+25.6 "Double Sever 2" sync /:Phantom Ray:F7C:/
+27.4 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+
+41.5 "Rapid Sever" sync /:Phantom Ray:F7A:/
+47.7 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+52.5 "Double Sever 1" sync /:Phantom Ray:F7B:/
+56.5 "Double Sever 2" sync /:Phantom Ray:F7C:/
+58.3 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/ window 8,10 jump 27.4
+
+72.4 "Rapid Sever" sync /:Phantom Ray:F7A:/ jump 41.5
+78.6 "Atmospheric Displacement"
+83.3 "Double Sever 1"
+87.4 "Double Sever 2"
+89.2 "Atmospheric Displacement"
+#Phase 2 at < 75% HP
+
+97.7 "--sync--" #landing for phase 2 loop
+100.0 "Damage Up" sync /:Phantom Ray:F7F:/  window 100,30 #Overclock
+104.8 "Double Sever 1" sync /:Phantom Ray:F7B:/
+107.6 "Atmospheric Compression" sync /:Phantom Ray:F80:/
+109.0 "Double Sever 2" sync /:Phantom Ray:F7C:/
+111.8 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+118.9 "Rapid Sever" sync /:Phantom Ray:F7A:/
+124.0 "Rapid Sever" sync /:Phantom Ray:F7A:/
+128.2 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+133.1 "Double Sever 1" sync /:Phantom Ray:F7B:/
+137.3 "Double Sever 2" sync /:Phantom Ray:F7C:/
+139.0 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+146.1 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/ jump 97.7
+
+148.4 "Damage Up" sync /:Phantom Ray:F7F:/ jump 100.0
+153.2 "Double Sever 1"
+155.8 "Atmospheric Compression"
+157.4 "Double Sever 2"
+160.2 "Atmospheric Displacement"
+167.4 "Rapid Sever"
+172.5 "Rapid Sever"
+176.7 "Atmospheric Displacement"
+
+###Minotaur
+1000.0 "--start--" sync /:The high-level incubation bay will be sealed off/ window 1999,5
+1000.0 "Start"
```

Comment:
This is a tiny nitpick, but no need for "Start" anywhere, and also maybe prefer "--sync--" over "--start--" just to match other timelines? I doesn't really matter, but it's mostly just a consistency thing.